solidity/random-beacon/test/fixtures/index.ts

```diff
@@ -172,6 +181,23 @@ export async function testDeployment(): Promise<DeployedContracts> {
   return { ...contracts, ...newContracts }
 }
 
+async function updateTokenStakingParams(
+  t: T,
+  staking: TokenStaking,
+  deployer: SignerWithAddress
+) {
+  // initialNotifierTreasury should be configured high enough to execute all the
+  // slashing in test suites.
+  const initialNotifierTreasury = to1e18(9_000_000) // 9MM T
```

Comment:
Today I have learned that MM in accounting is one million. Thank you @nkuba, thank you Google.

Reply:
🤓